UnityProject/Assets/Scripts/Medical/BodyBag.cs

```diff
@@ -0,0 +1,48 @@
+using Mirror;
+using UnityEngine;
+
+public class BodyBag : NetworkBehaviour, IInteractable<MouseDrop>
```

Comment:
wait, it looks like you're running this code on client side only, like this:
https://github.com/unitystation/unitystation/wiki/Implementing-Interactions#i-dont-need-any-networking-i-just-want-client-side-interaction-logic
it's only suitable if you call Cmd or send a special message (a niche use case).
I think you should extend `Interactable<MouseDrop>` here instead, move validation part to to `WillInteract` and call TryFoldUp within `ServerPerformInteraction`.

Reply:
actually I fell in the same trap the first time I made an item using IF2

Reply:
Updated to use this, need to test and will push up in new commit

Reply:
Pushed up!